app/views/layouts/application.html.erb

```diff
@@ -12,6 +12,12 @@
   </head>
 
   <body>
+    <nav class="crumbs">
+        <ul>
+            <li class="crumb"><a href="#">Rumblr</a></li>
+            <li class="crumb"><%= link_to 'Sign in', sign_in_path %></li>
```

Comment:
How do you feel about adding these strings to the locale files? When you run the specs, you'll find out where to store the values.

```suggestion
            <li class="crumb"><a href="#"><%= t(".title") %></a></li>
            <li class="crumb"><%= link_to t(".sign_in"), sign_in_path %></li>
```